index.html

```diff
@@ -14,46 +14,258 @@
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javaScript</button></li>
+        </ul>
+
+        <button class="btn">See Project</button>
+      </section>
+
+      <!-- multipost section -->
+      <section class="post-section">
+        <div class="postsection-image"></div>
+
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javaScript</button></li>
+        </ul>
+
+        <button class="btn">See Project</button>
+      </section>
+
+      <!-- stories section -->
+      <section class="stories-section">
+        <div class="storysection-image"></div>
+        <h2 class="multi-Post">
+          Multi-Post<br />
+          Stories
+        </h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="button-container">
+          <button class="html">html</button>
+          <button class="css">css</button>
+          <button class="javascript">javaScript</button>
+        </div>
+
+        <button class="btn">See Project</button>
+      </section>
+
+      <section class="about-section">
+        <h2 class="myself">
+          About<br />
+          Myself
+        </h2>
+        <p class="career">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h4 class="connect">LET’S CONNECT</h4>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <button class="resume">Get my resume</button>
```

Comment:
- [ ] You have done a great job following the Figma design, but I think you missed implementing the exact button interactions. The reason we are emphasising on getting the design exact is because this is what you will face at your job. Currently, your `Hover` gives a different background color and when the button is `Pressed` it does not give the background color as per the Figma template you have chosen. See screenshot below of button interactions colors and kindly implement them.
**Note:** Please apply this feedback to all the other buttons of this section or milestone. 
![image](https://user-images.githubusercontent.com/62849872/143722203-841693f9-1271-4e21-9217-9b85ee9fad69.png)